spa/src/components/modals/PersonalInfoModal.vue

```diff
@@ -113,6 +118,9 @@ export default Vue.extend({
 	  firstName: this.info.firstName,
 	  lastName: this.info.lastName,
         });
+        this.$http.post("/member/updateinfo", {
```

Comment:
Is there a way to move this to one update call instead of having one for name and one for chat?

Reply:
done